docs/output.md

```diff
@@ -6,554 +6,671 @@ This document describes the output produced by the pipeline. Most of the plots a
+</details>
 
-### MSI status
+### Microsatellite instability
 
 [Microsatellite instability](https://en.wikipedia.org/wiki/Microsatellite_instability) is a genetic condition associated to deficiencies in the mismatch repair (MMR) system which causes a tendency to accumulate a high number of mutations (SNVs and indels).
 An altered distribution of microsatellite length is associated to a missed replication slippage which would be corrected under normal MMR conditions.
 
-#### MSIsensor
+#### MSIsensorPro
 
-[MSIsensor](https://github.com/ding-lab/msisensor) is a tool to detect the MSI status of a tumor scanning the length of the microsatellite regions.
-It requires a normal sample for each tumour to differentiate the somatic and germline cases.
+[MSIsensorPro](https://github.com/xjtu-omics/msisensor-pro) is a tool to detect the MSI status of a tumor scanning the length of the microsatellite regions.
+It requires a normal sample for each tumour to differentiate the somatic and germline cases. For further reading see the [MSIsensor paper](https://www.ncbi.nlm.nih.gov/pubmed/24371154).
 
-For a Tumor/Normal pair:
+<details markdown="1">
+<summary>Output files for tumor/normal paired samples</summary>
 
-**Output directory: `results/VariantCalling/[TUMORSAMPLE]_vs_[NORMALSAMPLE]/MSIsensor`**
+**Output directory: `{outdir}/variantcalling/<tumorsample_vs_normalsample>/msisensor`**
 
-- `[TUMORSAMPLE]_vs_[NORMALSAMPLE]_msisensor`
+- `<tumorsample_vs_normalsample>`
   - MSI score output, contains information about the number of somatic sites.
-- `[TUMORSAMPLE]_vs_[NORMALSAMPLE]_msisensor_dis`
+- `<tumorsample_vs_normalsample>_dis`
   - The normal and tumor length distribution for each microsatellite position.
-- `[TUMORSAMPLE]_vs_[NORMALSAMPLE]_msisensor_germline`
+- `<tumorsample_vs_normalsample>_germline`
   - Somatic sites detected.
-- `[TUMORSAMPLE]_vs_[NORMALSAMPLE]_msisensor_somatic`
+- `<tumorsample_vs_normalsample>_somatic`
   - Germline sites detected.
-
-For further reading see the [MSIsensor paper](https://www.ncbi.nlm.nih.gov/pubmed/24371154).
+  </details>
 
 ## Variant annotation
 
-This directory contains results from the final annotation steps: two tools are used for annotation, [snpEff](http://snpeff.sourceforge.net/) and [VEP](https://www.ensembl.org/info/docs/tools/vep/index.html).
-Only a subset of the `VCF` files are annotated, and only variants that have a `PASS` filter.
-Currently, `FreeBayes` results are not annotated as we are lacking a decent somatic filter.
+This directory contains results from the final annotation steps: two tools are used for annotation, [snpEff](http://snpeff.sourceforge.net/) and [VEP](https://www.ensembl.org/info/docs/tools/vep/index.html). Both results can also be combined by setting `--tools merge`.
+All variants present in the called VCF files are annotated. For some variantcallers this can mean that the variants are already filtered by `PASS`, for some this needs to be done during post-processing.
```

Comment:
```suggestion
All variants present in the called VCF files are annotated. For some variant callers this can mean that the variants are already filtered by `PASS`, for some this needs to be done during post-processing.
```